doc/source/train/user-guides/persistent-storage.rst

```diff
@@ -1,62 +1,370 @@
-Configuration and Persistent Storage
-====================================
-
 .. _persistent-storage-guide:
 
-.. _train-run-config:
+.. _train-log-dir:
+
+Configuring Persistent Storage
+==============================
+
+A Ray Train run produces results, checkpoints, and other artifacts.
```

Comment:
Would be good to introduce the definitions for these different concepts here, so readers know how to think about them throughout the rest of the document